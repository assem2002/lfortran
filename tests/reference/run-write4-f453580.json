{
    "basename": "run-write4-f453580",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/write4.f90",
    "infile_hash": "3ea1b036446d9c3c9948c04d5b3bba170d5163d80d23e55ae2249956",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-write4-f453580.stdout",
    "stdout_hash": "ef442c30b6e21ccbe9b3e23e7ab441b37638b3d3a5be4f42f9a7a00a",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}