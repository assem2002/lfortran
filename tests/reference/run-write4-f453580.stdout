hi,  @how are you?
I  @am( @doing 1 @good= @